tests/cli_ui_test.py

```diff
@@ -36,136 +36,189 @@
                     "message": "Please write down your description:",
                 },
             ],
-            {"prompt_ret": {"wants_to_document_model": False}},
+            {"wants_to_document_model": False},
+            {"wants_to_document_model": False},
+            id="no_document_model",
         ),
-        (
-            "model",
-            [
-                {
-                    "type": "confirm",
-                    "name": "wants_to_document_model",
-                    "message": "Model Description: This is my previously documented model. Document?",
-                    "default": True,
-                },
-                {
-                    "type": "text",
-                    "name": "model_description",
-                    "message": "Please write down your description:",
-                },
-            ],
-            {"prompt_ret": {"wants_to_document_model": True, "model_description": ""}},
-        ),
-        (
-            "undocumented_columns",
+    ],
+)
+def test__document_model(mocker, question_payload, questionary_outputs, expected_results):
+    from dbt_sugar.core.ui.cli_ui import UserInputCollector
+
+    print(question_payload)
```

Comment:
do you want to keep the print here?

Reply:
Good catch I will remove it (as well as the part that is commented out).